src/browser-polyfill.js

```diff
@@ -348,10 +348,10 @@ if (typeof browser === "undefined") {
       },
     };
 
-    // Create an object that has the real target as its prototype
+    // Create a new empty object and copy the properties of the original chrome object
     // to prevent a Proxy violation exception for the devtools API getter
     // (which is a read-only non-configurable property on the original target).
-    const targetObject = Object.create(chrome);
+    const targetObject = Object.assign({}, chrome);
```

Comment:
Would it help to do `Object.freeze(Object.assign({}, chrome))` to make it read-only? I may not be understanding that part right though.

Reply:
I'm not sure that Object.freeze the target object is really useful (it is also never exposed outside of the Proxy instance that uses it), I'm going to take a look at it once we have the smoke test (#66) in place.